test/unit/on-attach.spec.js

```diff
@@ -749,4 +771,64 @@ describe('onAttach', function() {
       });
     });
   });
+
+  describe('when showing an empty CollectionView', function() {
```

Comment:
I think we are doing that with `this.region.show(this.collectionView)`, because `this.region` is [attached to a fixture](https://github.com/ianmstew/backbone.marionette/blob/collection-view-attach/test/unit/on-attach.spec.js#L6-L9).